src/data/hr.prepare.json

```diff
@@ -184,11 +186,23 @@
                 "data_delimiter_regexp": "\\.\n",
                 "copy_query_regex": "COPY.*FROM stdin;"
             },
-            "destination": "hr.kvkhdnm00",
-            "id": "import_kvkhdn_sql",
             "depends_on": [
                 "clear_schemas"
             ]
+        },
+        {
+            "type": "execute_sql",
+            "description": "Create HR vestigingen table",
+            "id": "create_hr_ves",
+            "query_src": "file",
+            "query": "data/sql/hr/create.vestigingen.sql",
+            "depends_on": [
+                "clear_schemas",
+                "import_kvkves",
+                "import_kvkbeshdn",
```

Comment:
Ik zou `import_kvkbeshdn` hernoemen naar `import_kvkveshdnm` (2x) omdat het om de `kvkveshdnm00` tabel gaat.

En ik mis hier `import_kvkhdn` nog in `depends_on`.

Reply:
Done. maar ik heb "import_kvkveshdn" van gemaakt!